src/frontend-samples/scientific-visualization-sample/AnimationApi.ts

```diff
@@ -0,0 +1,165 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Please rename the files and main classes.  I propose we use the name "ScientificViz"... for example, "AnimationApi" should be "ScientificVizApi", AnimationWidget => ScientificVizWidget, etc.

Reply:
Done